source/_integrations/nest.markdown

```diff
@@ -184,6 +184,8 @@ and typically looks like `projects/sdm-prod/topics/EXAMPLE`.
 
 1. Lower the message retention duration to be something short (e.g., 10 minutes or under an hour) to avoid a large backlog of updates when Home Assistant is turned off.
 
+1. Select **Never expire** as the *Expiry Period*. To prevent the subscription you're creating being removed if for example your Home Assistant or the integration is offline for a while.
```

Comment:
```suggestion
1. Select **Never expire** as the *Expiry Period* to prevent the subscription you're creating being removed if for example your Home Assistant or the integration is offline for a while.
```